rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java

```diff
@@ -0,0 +1,59 @@
+package co.rsk.config;
+
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class BridgeConstantsTest {
+
+    @Test
+    void test_getFundsMigrationAgeSinceActivationEnd() {
```

Comment:
We should repeat this tests for each network, just to be sure. Seems like a good opportunity to make use of parameterized tests

Reply:
That's a fantastic idea!